src/backend/base/langflow/schema/playground_events.py

```
@@ -170,12 +170,13 @@ def create_token(chunk: str, id: str) -> TokenEvent:  # noqa: A002
 }
 
 
-def create_event_by_type(
-    event_type: Literal["message", "error", "warning", "info", "token"], **kwargs
-) -> PlaygroundEvent | dict:
+def create_event_by_type(event_type: str, **kwargs) -> PlaygroundEvent | dict:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)